simcon_project/users/models/assignment.py

```diff
@@ -1,5 +1,13 @@
 from django.db import models
 import uuid
+from json import JSONEncoder
+from uuid import UUID
+
+JSONEncoder_olddefault = JSONEncoder.default
+def JSONEncoder_newdefault(self, o):
+    if isinstance(o, UUID): return str(o)
+    return JSONEncoder_olddefault(self, o)
+JSONEncoder.default = JSONEncoder_newdefault
```

Comment:
Can you explain what this does? I have no clue what it's used for

Reply:
This solves an issue I was having with passing the assignment id through the URL to conversation-start. Not sure why it was bugging out on assignment and not other uuids.

Here's a link that describes it a bit:
https://arthurpemberton.com/2015/04/fixing-uuid-is-not-json-serializable

Reply:
That's a weird bug, but now I will know where to look if I run into the same problem